src/ToDo/Presentation/Dialogs/ExpirationDateViewModel.cs

```diff
@@ -1,6 +1,41 @@
 namespace ToDo.Presentation.Dialogs;
 
+[ReactiveBindable]
 public partial class ExpirationDateViewModel
 {
-    
+	private readonly INavigator _navigator;
+
+	public DateTime DueDate
+	{
+		get;
+		set;//TODO: the idea is return to TaskViewModel with the DueDate value / call BackToTaskMethod
+	}
+	private ExpirationDateViewModel(INavigator navigator)
+	{
+		_navigator = navigator;
+	}
+
+	private async ValueTask BackToTask()
+	{
+		 await _navigator.NavigateBackWithResultAsync(this, data: new DateTimeData { DateTime = DueDate });
+	}
+
+	public ICommand SelectToday => Command.Async(DoSelectToday);
+	private async ValueTask DoSelectToday(CancellationToken ct)
```

Comment:
You no longer need to explicitly declare the `ICommand` when your method is public and has at most 1 parameter (the `CommandParameter`) + an optional cancellation token.
```suggestion
	public async ValueTask SelectToday(CancellationToken ct)
```